### PR TITLE
Fix markup with cartoassets

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam could you take a look?

- Fix markup for consistence 
- Fix map title using ellipsis.

image

### CODE REVIEW

File: app/assets/stylesheets/editor-3/header.css.scss [hunk 1]
```diff
@@ -36,11 +36,14 @@
 }
 .Editor-HeaderInfo-titleText {
   @include flex(1);
+  display: block;
   width: 0;
   padding-right: 16px;
   text-overflow: ellipsis;
   white-space: nowrap;
   overflow: hidden;
+  -webkit-line-clamp: 1;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/header.css.scss]: Only for webkit? Be careful with this one, I would use that kind of things, if it is available for all browsers -> @mixin and use it accordingly.

**Comment by xavijam** [comment]
Apart from the comment (that I consider important) 👍

**Comment by xavijam** [comment]
Ping!

**Comment by piensaenpixel** [comment]
@xavijam fixed sorry
image

**Comment by xavijam** [comment]
:+1:

**Comment by piensaenpixel** [comment]
retest

File: app/assets/stylesheets/editor-3/header.css.scss [hunk 2]
```diff
@@ -36,6 +36,7 @@
 }
 .Editor-HeaderInfo-titleText {
   @include flex(1);
+  display: block;
   width: 0;
   padding-right: 16px;
   text-overflow: ellipsis;
```
Comment (xavijam) [inline app/assets/stylesheets/editor-3/header.css.scss]: hey, why don't we use `u-ellipsis` class for that?

**Pull request merged**